paper/basic_training.tex

```diff
@@ -225,9 +225,10 @@ \subsubsection{Key concepts}
 In this sense, the laws of thermodynamics provide us rigorous sanity checks in addition to many useful mathematical relations for computing properties.
 Basic thermodynamic principles thus also dictate proper simulation protocols and associated best practices.
 
-The concept of the thermodynamic limit is particularly important.
+The concept of the thermodynamic limit is particularly important here.
 Specifically, as the size of a finite system is increased, keeping the particle number density roughly constant, at some point it is said to reach the thermodynamic limit where its behavior is bulk-like and no longer depends on the extent of the system.
-Thus, small systems will exhibit unique behaviors that reflect their microscopic size, but sufficient large systems are said to have reached the thermodynamic limit and macroscopic thermodynamics applied.
+Thus, small systems will exhibit unique behaviors that reflect their microscopic size, but sufficiently large systems are said to have reached the thermodynamic limit and macroscopic thermodynamics applied.
+This is due to the fact that the effect of interfaces or boundaries have largely been removed, and that averages of system properties are now over a sufficiently large number of molecules that any instantaneous snapshot of the system roughly corresponds to average behavior (i.e. fluctuations in system properties become negligible).
```

Comment:
Also central limit theorem issues; not sure if we need to mention, but it's not JUST a boundary effect.

Reply:
I tried to mention this with the idea that you average over more molecules and reduce fluctuations in properties with system size.  Updated to make more clear, but it may still need some work.